static/js/services/enketo.js

```diff
@@ -21,6 +22,7 @@ angular.module('inboxServices').service('Enketo',
     var objUrls = [];
     var xmlCache = {};
     var FORM_ATTACHMENT_NAME = 'xml';
+    var REPORT_ATTACHMENT_NAME = this.REPORT_ATTACHMENT_NAME = 'content';
```

Comment:
What is `this` here, the service?  Presumably the inner var is to make references simpler within the service's code.